openxla_patches/gpu_race_condition.diff

```diff
@@ -1,14 +0,0 @@
-diff --git a/xla/service/gpu/gpu_executable.cc b/xla/service/gpu/gpu_executable.cc
-index 9279bd877..fab926a7c 100644
---- a/xla/service/gpu/gpu_executable.cc
-+++ b/xla/service/gpu/gpu_executable.cc
-@@ -669,8 +669,7 @@ absl::StatusOr<ExecutionOutput> GpuExecutable::ExecuteAsyncOnStreamImpl(
- #endif  // GOOGLE_CUDA || TENSORFLOW_USE_ROCM
-
-   // Force synchronous execution if the allocator requires it.
--  const bool block_host_until_done =
--      !memory_allocator->AllowsAsynchronousDeallocation();
-+  const bool block_host_until_done = true;
```

Comment:
this one is for a GPU repo, did openxla somehow fixed it?

Reply:
The CI shows the GPU test pass, will this be a concern? Just in case the CI doesn't catch the issue.

Reply:
it was another user that complains about some inconisistnet behavior if this is enabled, I would just leave it here if nothing changed in openxla side.

Reply:
Got it, I will just not bother it.